astropy/coordinates/baseframe.py

```diff
@@ -810,11 +810,13 @@ def represent_as(self, new_representation, in_frame_units=False):
         >>> from astropy.coordinates import SkyCoord, CartesianRepresentation
         >>> coord = SkyCoord(0*u.deg, 0*u.deg)
         >>> coord.represent_as(CartesianRepresentation)
-        <CartesianRepresentation x=1.0, y=0.0, z=0.0>
+        <CartesianRepresentation (x, y, z) in
```

Comment:
Not really a problem with this PR, but strange not to have a unit here -- either add "dimensionless units" or maybe just remove "in"?